contracts/cw20-escrow/src/contract.rs

```diff
@@ -192,17 +192,17 @@ pub fn try_refund<S: Storage, A: Api, Q: Querier>(
         Err(StdError::unauthorized())
     } else {
         // we delete the escrow (TODO: expose this in Bucket for simpler API)
-        prefixed(PREFIX_ESCROW, &mut deps.storage).remove(id.as_bytes());
+        prefixed(&mut deps.storage, PREFIX_ESCROW).remove(id.as_bytes());
```

Comment:
Note, this is now in `Bucket.remove` you can use that now.

Reply:
Will change those in another branch / PR.